src/content/guides/asset-management.md

```diff
@@ -121,20 +121,19 @@ Now run your build command:
 ``` bash
 npm run build
 
-Hash: 9a3abfc96300ef87880f
```

Comment:
Hi @venelinpetrov 
To reduce the future maintance hussle, it was decided to only touch output when its `assets` information gets outdated. Also we will not show the rest of the output, only the assets information.

Please update your PR to only leave the assets info in the output logs like this:

```
...
                               Asset      Size  Chunks                    Chunk Names
da4574bb234ddc4bb47cbe1ca4b20303.png  3.01 MiB          [emitted]  [big]
                           bundle.js  76.7 KiB       0  [emitted]         main
Entrypoint main = bundle.js
...
```

Sorry if my example is misaligned.

Reply:
Let's merge this one as it is. We need to change all examples in one PR and then force it later.

Reply:
ok